packages/utils/src/tests/test_checkpoint.cairo

```diff
@@ -0,0 +1,109 @@
+mod tests {
```

Comment:
```suggestion
```

Reply:
whoops. I was originally adding tests inside the same checkpoint file and then decided to improve upon our previous design and moved it to it's onw file but forgot to get rid of the test module